lib/themes/dosomething/paraneue_dosomething/templates/reportback/reportback-simple-permalink.tpl.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * @file
+ * Reportback confirmation/permalink page for Rogue.
+ */
+?>
+<article class="reportback__permalink">
+  <header role="banner" class="header ">
+    <div class="wrapper">
+      <h1 class="header__title">Awesome! We got your photo.</h1>
+      <p class="header__subtitle">You've entered into the scholarship and your photo will show up in the gallery soon!</p>
```

Comment:
ah ok! I was just going off the copy from the wire but good catch! 

@ngjo is there alternative copy to have here if there is no scholarship? Maybe just "Your photo will show up in the gallery soon!"?